extensions/2.0/Khronos/KHR_materials_pbrSpecularGlossiness/README.md

```diff
@@ -101,7 +101,7 @@ The RGBA components of the reflected diffuse color of the material. Metals have
 
 ### diffuseTexture
 
-The diffuse texture. This texture contains RGB(A) components of the reflected diffuse color of the material in sRGB color space. If the fourth component (A) is present, it represents the alpha coverage of the material. Otherwise, an alpha of 1.0 is assumed. The `alphaMode` property specifies how alpha is interpreted. The stored texels must not be premultiplied.
+The diffuse texture. This texture contains RGB components of the reflected diffuse color of the material encoded with sRGB transfer function. If the fourth component (A) is present, it represents the linear alpha coverage of the material. Otherwise, an alpha of 1.0 is assumed. The `alphaMode` property specifies how alpha is interpreted. The stored texels must not be premultiplied.
```

Comment:
Each place we use "encoded with sRGB transfer function" should change to "encoded with the sRGB transfer function".

Reply:
Done.